docs/developers/react-development-guidelines.md

```diff
@@ -0,0 +1,397 @@
+---
+id: react-development-guidelines
+title: React Development Guidelines
+---
+
+[comment]: # (mx-abstract)
+
+## Introduction
+
+Every developer has his/her own code style that has been developed along the way, with bits and quirks that, at some point, become a part of oneself. 
+
+However, in a big team and in a big project, small quirks and personal preferences can add up and turn the codebase into a big lasagna. 
+
+Given this, we have established some basic principles and a code style we would like to follow in this project. These are, of course, not set in stone, and can be changed, given a valid reason.
```

Comment:
"to follow in this project" ... which project ? Maybe this can be reformulated to be more generic